src/commands/bindle.rs

```diff
@@ -89,6 +87,24 @@ pub struct Push {
     )]
     pub bindle_server_url: String,
 
+    /// Basic http auth username for the bindle server
+    #[structopt(
+        name = BINDLE_USERNAME,
+        long = "bindle-username",
+        env = BINDLE_USERNAME,
+        requires(BINDLE_PASSWORD)
```

Comment:
Nit: can we not write `requires = ...` for consistency?  I thought this worked elsewhere but could be wrong.

Reply:
Oh certainly, good call.

Reply:
Updated in https://github.com/fermyon/spin/pull/407/commits/6311a2def0e4ea2db3837ced662558b6cc75846b